packages/app/src/queries/futures/useLeaderboard.ts

```diff
@@ -36,11 +36,11 @@ export const DEFAULT_LEADERBOARD_DATA = {
 	all: [],
 }
 
-const useLeaderboard = (searchTerm: string, options?: UseQueryOptions<LeaderboardResult>) => {
+const useLeaderboard = (searchTerm: string, options?: UseQueryOptions<any>) => {
```

Comment:
What's the reason we relaxed the type here? We should try and avoid the use of any and always be strict with types